src/txadmission.cpp

```diff
@@ -540,7 +540,9 @@ bool ParallelAcceptToMemoryPool(Snapshot &ss,
 
     const uint32_t cds_flag =
         IsNov152018Enabled(chainparams.GetConsensus(), chainActive.Tip()) ? SCRIPT_ENABLE_CHECKDATASIG : 0;
-    const uint32_t flags = STANDARD_SCRIPT_VERIFY_FLAGS | cds_flag;
+    const uint32_t svflag =
+        IsSv2018Enabled(chainparams.GetConsensus(), chainActive.Tip()) ? SCRIPT_ENABLE_MUL_SHIFT_INVERT_OPCODES : 0;
+    const uint32_t flags = STANDARD_SCRIPT_VERIFY_FLAGS | cds_flag | svflag;
```

Comment:
Shouldn't this also make a check for IsSv2018Scheduled()?

Reply:
Its not needed.  Sv2018 cannot be "enabled" if it is not also "scheduled".  But the opposite is not true... it can be "scheduled" but not "enabled".  So you really only need to use "scheduled" when checking that case.

Reply:
The way I read it now, since we won't be scheduling the svfork by default then the mining fork time for SV will be 0..that means that IsEnabled() will always return true , essentially the new opcodes will be active at all times, no?

Reply:
Looks to me like also the same line above for checkdatasig?  Hmm, am i just not getting this, it looks to me like both forks will be active here and we'll be allowing all txns for the ABC and SV forks into the mempool after the fork happens, regardless of whether they were scheduled or not...

Reply:
zero is special-cased to always mean NOT active:

bool CBlockIndex::IsforkActiveOnNextBlock(int time) const
{
    if (time == 0)
        return false;
....

Reply:
ah, i didn't know that was there...thx :)